### PR TITLE
CLDR-17416 v45 BRS validity

### PR DESCRIPTION
**Pull request by srl295**
- also note we are starting to see spdx changes from CLDR-17110

CLDR-17416

- [ ] This PR completes the ticket.

ALLOW_MANY_COMMITS=true

### CODE REVIEW

**Comment by macchiati** [comment]
Tests failing

File: common/validity/script.xml [hunk 1]
```diff
@@ -35,12 +35,12 @@
 			Vaii Vith
 			Wara Wcho
 			Xpeo Xsux
-			Yezi Yiii Zanb
```
Comment (srl295) [inline common/validity/script.xml:38]: what's going on with `Zanb`? @markusicu @macchiati

Comment (srl295) [reply common/validity/script.xml:38]: We've been here before.. https://github.com/unicode-org/cldr/pull/3233#issuecomment-1701848571 

blocked by CLDR-17056  - I'll back out `Zanb` for now.

File: common/validity/script.xml [hunk 2]
```diff
@@ -35,7 +35,7 @@
...
+			Yezi Yiii Zanb <!-- Note: for Zanb see CLDR-17056 -->
```
Comment (srl295) [inline common/validity/script.xml:38]: ^ so that we see this the next time the generator is run

**Review by macchiati — CHANGES_REQUESTED** [review]
I decided it was time to fix the tooling. I figured that rather than have the tooling be a separate PR, it was better to do it all at once. So could you review https://github.com/unicode-org/cldr/pull/3537 and if that's ok, merge it and close this PR?

**Pull request closed without merging**